fuzzing_setup.sh

```diff
@@ -20,4 +20,4 @@ done_text="$(tput bold)DONE.$(tput sgr0)"
 set -e
 
 # Install cargo-fuzz library.
-cargo +stable install cargo-fuzz
+cargo +stable install cargo-fuzz --version 0.10.0
```

Comment:
I think there is a version 0.10.2 that also worked for me. Can you double check?

Reply:
Updated to 0.10.2